includes/sanitizers/class-amp-core-theme-sanitizer.php

```diff
@@ -811,10 +825,6 @@ static function() {
 				.featured-media amp-img {
 					position: static;
 				}
-
-				.wp-block-image img {
-					display: block;
-				}
```

Comment:
This rule is redundant since d976798 added a `amp-img:not([_]) { display: block }` style which also applies to this image.